runs/run_benchmark.py

```diff
@@ -28,10 +29,29 @@
     "n_ts_features": 9,
     "n_static_features": 1,
     "hidden_size": 32,
-    "num_layers": 3,
+    "num_layers": 1,
 }
 
 _BASELINE_MODEL_FIT_KWARGS = defaultdict(dict)
+_BASELINE_MODEL_FIT_KWARGS["LSTM"] = {
+    'batch_size': 32,
+    'num_epochs': 5,
+    'device': 'cuda' if torch.cuda.is_available() else 'cpu',
+    'optim_fn': torch.optim.Adam,
+    'scheduler_fn': torch.optim.lr_scheduler.StepLR,
+    'scheduler_kwargs': {"step_size": 2, "gamma": 0.8},
+    'val_fraction': 0.1,
+
+    'optimize_hyperparameters': True,
+    'param_space': {
+        'optim_kwargs': {
+            "lr": [0.01, 0.001],
```

Comment:
`lr`: I think the values are two large. Suggest 1e-4 and 1e-5.

Reply:
I'll do runs with lower lr's and more epochs today.